server/db/index.js

```diff
@@ -32,6 +32,7 @@ const Cart_Coffee = db.define('cart_coffee', {
 
 Cart.belongsToMany(Coffee, { through: Cart_Coffee});
 Coffee.belongsToMany(Cart, { through: Cart_Coffee});
+// not sure what is happening here
```

Comment:
This is the super many to many relationship I was talking about. Had to add this to be able to pull from Cart_Coffee in Sequelize, as w/o, it's an association and a lot more complicated.